### ISSUE TITLE
github.com/glebarez/sqlite@v1.4.8/sqlite.go:101:22: cannot convert 0 (untyped int constant) to *int

### ISSUE THREAD

**Issue by jianka** [glebarez/sqlite#62]

**Comment by Kqzz** [comment]
Currently having these issues too

```
# github.com/glebarez/sqlite
/home/kqzz/go/pkg/mod/github.com/glebarez/sqlite@v1.4.8/sqlite.go:101:22: cannot convert 0 (untyped int constant) to *int
/home/kqzz/go/pkg/mod/github.com/glebarez/sqlite@v1.4.8/sqlite.go:102:24: cannot convert 0 (untyped int constant) to *int
/home/kqzz/go/pkg/mod/github.com/glebarez/sqlite@v1.4.8/sqlite.go:103:21: cannot use -1 (untyped int constant) as *int value in assignment
/home/kqzz/go/pkg/mod/github.com/glebarez/sqlite@v1.4.8/sqlite.go:106:39: cannot use limit.Limit (variable of type *int) as type int in argument to strconv.Itoa

```

**Comment by glebarez** [comment]
What version of GORM are you using?

**Comment by Kqzz** [comment]
> What version of GORM are you using?

`gorm v1.24.0`

**Comment by glebarez** [comment]
This was fixed in [v1.5.0](https://github.com/glebarez/sqlite/releases/tag/v1.5.0) by merging the upstream (they already fixed this).
Please get latest version in your go.mod

**Comment by Kqzz** [comment]
Tysm!
